ProjectSettings/DynamicsManager.asset

```diff
@@ -12,4 +12,4 @@ PhysicsManager:
   m_SolverIterationCount: 6
   m_QueriesHitTriggers: 1
   m_EnableAdaptiveForce: 0
-  m_LayerCollisionMatrix: ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
```

Comment:
We like to not use layers in our core assets, since that can cause incompatibilities with developers projects.  We might need to brainstorm an alternate solution.